matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/permalinks/PermalinkService.kt

```diff
@@ -19,53 +19,59 @@ package org.matrix.android.sdk.api.session.permalinks
 import org.matrix.android.sdk.api.session.events.model.Event
 
 /**
- * Useful methods to create Matrix permalink (matrix.to links).
+ * Useful methods to create permalink (like matrix.to links or client permalinks).
  */
 interface PermalinkService {
 
     companion object {
         const val MATRIX_TO_URL_BASE = "https://matrix.to/#/"
+        const val MATRIX_TO_CUSTOM_SCHEME_URL_BASE = "element://"
```

Comment:
Mentions of "element" should not occur in the Matrix SDK. Is there an other solution?

Reply:
As I said, I rolled back this custom scheme management which was in the HomeActivity. That implies this kind of link will only work from those coming from a deep link, not for the other accessed from a room which are not passed to the HomeActivity (I kept the previous behaviour)

Reply:
Yes but still, we should avoid having reference to element in the SDK